Maple2.Server.Game/Trigger/TriggerContext.Player.cs

```diff
@@ -91,6 +91,11 @@ public void MoveUser(int mapId, int portalId, int boxId) {
             if (!Field.TryGetPortal(portalId, out FieldPortal? portal)) {
                 return;
             }
+
+            if (!Field.ValidPosition(portal.Position)) {
+                return;
+            }
+
             foreach (FieldPlayer player in PlayersInBox(boxId)) {
                 player.Session.Send(PortalPacket.MoveByPortal(player, portal));
             }
```

Comment:
Moving By Portal would always need to validate position, so if anywhere else comes up where we need to use said packet, we would need to validate the position. I feel like it'd be better suited if we make a function to combine this.